src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Core/StringInterpolationPropertyTests.fs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace FSharp.Core.Unittests.FSharp_Core.Microsoft_FSharp_Core
+
+open System
+open FSharp.Core.Unittests.LibraryTestFx
+open NUnit.Framework
+open FsCheck
+
+module PropertyTests =
```

Comment:
I think these test should be in Compiler unit tests.
FSharp.Core shouldn't know about string interpolation.